atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/thrift/SlicePredicates.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.keyvalue.cassandra.thrift;
+
+import java.nio.ByteBuffer;
+
+import org.apache.cassandra.thrift.SlicePredicate;
+import org.apache.cassandra.thrift.SliceRange;
+import org.immutables.value.Value;
+
+import com.palantir.atlasdb.encoding.PtBytes;
+import com.palantir.atlasdb.keyvalue.cassandra.CassandraKeyValueServices;
+
+public final class SlicePredicates {
```

Comment:
We should document that a SlicePredicate just limits the column selection of the query, as the documentation for the thrift api is lacking in this. We could do this by renaming all our `SlicePredicate` variables to `columnPredicates`, or encapsulating a SlicePredicate on a `ColumnSlicePredicate`. If we don't want to do either of those, we could add Javadocs to this class explaining this.